Core/clim-basic/windowing/events.lisp

```diff
@@ -134,6 +140,12 @@
    (graft-x :reader pointer-event-native-graft-x)
    (graft-y :reader pointer-event-native-graft-y) ))
 
+(defmethod print-object ((event pointer-event) stream)
+  (print-unreadable-object (event stream :type t :identity nil)
+    (format stream "X: ~d, Y: ~d"
```

Comment:
I know this is cosmetic but please make the format consistent (labels as keywords or with following colons, comma between fields or not).

Reply:
fixed. I took hint from regions.lisp, where
- if only pair of coordinates is printed, it is "~s ~s"
- when name val are intervened, it is "NAME VAL NAME VAL"
- when name coord are intervened, it is "START ~s:~s END ~s:~s"